projects/opendr_ws/src/opendr_perception/README.md

```diff
@@ -354,35 +354,39 @@ whose documentation can be found here: [Deep Sort docs](../../../../docs/referen
 An [image dataset node](#image-dataset-ros-node) is also provided to be used along these nodes.
 Make sure to change the default input topic of the tracking node if you are not using the USB cam node.
 
-### Panoptic Segmentation ROS Node
-
-You can find the panoptic segmentation ROS node python script [here](./scripts/panoptic_segmentation_efficient_ps_node.py) to inspect the code and modify it as you wish to fit your needs.
-The node makes use of the toolkit's [panoptic segmentation tool](../../../../src/opendr/perception/panoptic_segmentation/efficient_ps/efficient_ps_learner.py) whose documentation can be found [here](../../../../docs/reference/efficient-ps.md)
-and additional information about Efficient PS [here](../../../../src/opendr/perception/panoptic_segmentation/README.md).
-
-#### Instructions for basic usage:
-
-1. Start the node responsible for publishing images. If you have a USB camera, then you can use the `usb_cam_node` as explained in the [prerequisites above](#prerequisites).
-
-2. You are then ready to start the panoptic segmentation node:
-
-    ```shell
-    rosrun opendr_perception panoptic_segmentation_efficient_ps_node.py
-    ```
-
-    The following optional arguments are available:
-   - `-h or --help`: show a help message and exit
-   - `-i or --input_rgb_image_topic INPUT_RGB_IMAGE_TOPIC` : listen to RGB images on this topic (default=`/usb_cam/image_raw`)
-   - `-oh --output_heatmap_topic OUTPUT_HEATMAP_TOPIC`: publish the semantic and instance maps on this topic as `OUTPUT_HEATMAP_TOPIC/semantic` and `OUTPUT_HEATMAP_TOPIC/instance`, `None` to stop the node from publishing on this topic (default=`/opendr/panoptic`)
-   - `-ov --output_rgb_image_topic OUTPUT_RGB_IMAGE_TOPIC`: publish the panoptic segmentation map as an RGB image on this topic or a more detailed overview if using the `--detailed_visualization` flag, `None` to stop the node from publishing on this topic (default=`opendr/panoptic/rgb_visualization`)
-   - `--detailed_visualization`: generate a combined overview of the input RGB image and the semantic, instance, and panoptic segmentation maps and publish it on `OUTPUT_RGB_IMAGE_TOPIC` (default=deactivated)
-   - `--checkpoint CHECKPOINT` : download pretrained models [cityscapes, kitti] or load from the provided path (default=`cityscapes`)
```

Comment:
Similar to the previous comment, re-add these deleted lines and make any modification on top if you think it's necessary. The "instructions for basic usage" section is standard for all nodes in the document.